### DISCUSSION TITLE
Proposal: consider versioned documents

### DISCUSSION THREAD

**Discussion by cbl315** [fluxcd/flux2#1685] [Proposals] [answered]
Do we consider provider versioned documents? 
Just like `kubernetes`
![image](https://user-images.githubusercontent.com/6824860/128191109-f5bd1276-e3cb-40fb-b8fc-f84b552b800a.png)

**Comment by stefanprodan** [comment, marked as answer]
Yes once we hit GA aka flux 2.0.0
